protected_app_signals_cost.md

```diff
@@ -0,0 +1,163 @@
+PAS Cost Estimate Explainer
+
+**Authors:** 
+<br> [Stefano Galarraga][0], Google Privacy Sandbox 
+
+
+Overview and Goals
+==================
+
+[Protected App Signals][1] is a new design aimed at the app install ads market. The system is based on server-side execution via the B&A services and [TEE K/V services][2]. Shifting the auction execution on the server side has benefits in terms of latency and computing power but introduces cost to adtechs to run the cloud instances where B&A will reside.
+
+This document is based on the existing documentation about cost analysis and reduction ([B&A][3], K/V) and aims at providing an analysis of how to apply those concepts to the PAS architecture.
+
+Assumptions
+===========
+
+The following assumptions are made in computing the cost:
+
+*   About the K/V server used by the SFE we will consider two cases
```

Comment:
What are the 2 cases? This only elaborates 1 case? Maybe we missed a bullet here?